pkg/kubectl/cmd/edit.go

```diff
@@ -93,7 +93,7 @@ func NewCmdEdit(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		Example: fmt.Sprintf(editExample),
 		Run: func(cmd *cobra.Command, args []string) {
 			options.ChangeCause = f.Command(cmd, false)
-			if err := options.Complete(f, out, errOut, args); err != nil {
+			if err := options.Complete(f, out, errOut, args, cmd); err != nil {
```

Comment:
> I think include-uninitialized should be plumbed to every commands.

@caesarxuchao Currently we've only enabled `--include-uninitialized` in a few commands. To make it a global flag, we may need to make rules for every subcommand.